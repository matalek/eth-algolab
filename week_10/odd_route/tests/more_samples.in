10
2 1
0 1
0 1 2
3 4
0 2
0 1 1
1 0 2
0 2 2
0 2 11
10 18
4 8
0 3 2
0 6 1
4 3 4
7 4 10
9 0 6
9 3 8
9 1 4
2 3 6
2 6 10
2 5 10
3 0 6
3 4 5
3 1 9
1 0 8
1 1 7
1 5 9
5 6 4
5 5 6
10 21
3 2
3 5 6
3 1 5
3 7 4
6 8 4
2 2 7
2 9 1
2 7 8
4 7 10
5 3 3
5 2 9
5 4 2
9 2 4
9 9 9
7 6 7
7 4 6
8 3 8
8 8 8
0 4 9
0 5 7
0 9 3
0 8 5
10 23
0 6
3 7 5
7 7 2
7 4 6
7 5 1
7 1 8
8 6 2
8 4 1
8 0 1
6 3 3
6 6 3
6 2 5
6 0 7
4 8 7
4 6 1
4 4 4
9 7 6
9 8 4
9 9 6
9 2 6
2 5 3
0 5 3
5 2 10
1 5 5
10 28
1 7
6 4 1
6 2 2
6 8 6
9 3 4
4 2 10
4 0 3
7 2 7
7 8 3
2 4 7
2 2 6
2 5 2
0 9 7
0 4 1
3 2 8
3 0 6
3 3 8
3 1 9
3 5 2
8 6 5
8 7 6
8 3 1
1 9 7
1 0 8
5 6 10
5 4 8
5 8 9
5 1 4
5 5 4
10 30
4 3
9 9 10
9 0 6
9 7 4
9 5 1
8 2 1
8 6 5
0 9 1
0 0 4
0 3 9
0 2 4
3 9 9
3 7 9
3 2 2
3 6 10
3 1 4
7 8 6
7 5 2
2 7 10
2 6 7
2 4 1
6 1 3
1 0 4
1 7 6
1 2 3
1 1 4
4 9 7
4 3 5
5 7 4
5 2 1
5 4 9
10 22
3 3
5 8 1
5 9 4
6 5 2
6 2 4
6 3 10
8 5 2
8 4 4
8 7 9
8 2 4
7 2 7
7 3 10
7 1 1
2 6 5
2 2 4
3 5 7
3 1 9
3 0 2
9 4 1
9 8 4
9 2 10
1 3 8
1 9 1
10 23
3 2
4 4 10
4 5 5
3 5 10
3 2 2
5 9 1
5 8 5
5 2 8
0 2 5
9 0 10
9 9 7
9 7 10
6 5 4
6 7 2
8 4 7
7 1 3
1 8 10
1 7 10
1 1 4
2 4 8
2 3 8
2 0 10
2 6 3
2 2 2
10 23
4 1
8 8 2
8 0 4
8 3 6
0 9 8
9 8 3
9 7 6
2 2 4
2 6 7
2 7 1
2 4 9
5 9 6
5 3 3
5 4 7
3 3 6
1 5 8
1 6 10
1 1 6
7 9 8
7 7 8
4 0 1
4 2 9
4 6 1
4 3 9
