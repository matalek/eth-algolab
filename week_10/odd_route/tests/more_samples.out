no
5
no
27
no
37
5
27
17
no
